src/status_im/navigation2/core.cljs

```diff
@@ -13,8 +13,10 @@
                   :wallet      2
                   :browser     3})
 
-;; (defonce set-navigation-default-options
-;;   (.setDefaultOptions Navigation (clj->js {:options {:topBar {:visible false}}})))
+(defn change-root-status-bar-style [style]
+  (.mergeOptions Navigation
+                 (name :shell-stack)
```

Comment:
Can we write this as "shell-stack" .. As calling `name` or it will return the same result.

Reply:
done, thank you